src/mlpack/core/data/load_image_impl.hpp

```diff
@@ -90,7 +90,105 @@ bool Load(const std::vector<std::string>& files,
   return true;
 }
 
-} // namespace data
-} // namespace mlpack
+#ifdef HAS_STB
+
+inline bool LoadImage(const std::string& filename,
+                      arma::Mat<unsigned char>& matrix,
+                      ImageInfo& info,
+                      const bool fatal)
+{
+  unsigned char* image;
+
+  if (!ImageFormatSupported(filename))
+  {
+    std::ostringstream oss;
+    oss << "Load(): file type " << Extension(filename) << " not supported. ";
+    oss << "Currently it supports: ";
```

Comment:
```suggestion
    oss << "Currently it supports:";
```

Remove the extra space before the first supported file type.